.vscode/tasks.json

```diff
@@ -0,0 +1,156 @@
+            "command": "./run",
+            "args": ["build", "endoscopy_tool_tracking", "--type", "debug"],
+            "options": {
+                "cwd": "${env:WORKSPACE_DIR}"
+            },
+            "group": "build",
+            "problemMatcher": [],
+            "detail": "CMake template build task"
+        },
+        {
+            "type": "shell",
+            "label": "Build colonoscopy_segmentation",
+            "command": "./run",
+            "args": ["build", "colonoscopy_segmentation", "--type", "debug"],
+            "options": {
+                "cwd": "${env:WORKSPACE_DIR}"
+            },
+            "group": "build",
+            "problemMatcher": [],
+            "detail": "CMake template build task"
+        },
+        {
+            "type": "shell",
+            "label": "Build cuda_quantum",
+            "command": "./run",
+            "args": ["build", "cuda_quantum", "--type", "debug"],
+            "options": {
+                "cwd": "${env:WORKSPACE_DIR}"
+            },
+            "group": "build",
+            "problemMatcher": [],
+            "detail": "CMake template build task"
+        },
+        {
+            "type": "shell",
+            "label": "Build cvcuda_basic",
+            "command": "./run",
+            "args": ["build", "cvcuda_basic", "--type", "debug"],
+            "options": {
+                "cwd": "${env:WORKSPACE_DIR}"
+            },
+            "group": "build",
+            "problemMatcher": [],
+            "detail": "CMake template build task"
+        },
+        {
+            "type": "shell",
+            "label": "Build endoscopy_depth_estimation",
+            "command": "./run",
+            "args": ["build", "endoscopy_depth_estimation", "--type", "debug"],
+            "options": {
+                "cwd": "${env:WORKSPACE_DIR}"
+            },
+            "group": "build",
+            "problemMatcher": [],
+            "detail": "CMake template build task"
+        },
+        {
+            "type": "shell",
+            "label": "Build endoscopy_tool_tracking",
+            "command": "./run",
+            "args": ["build", "endoscopy_tool_tracking", "--type", "debug"],
+            "options": {
+                "cwd": "${env:WORKSPACE_DIR}"
+            },
+            "group": "build",
+            "problemMatcher": [],
+            "detail": "CMake template build task"
+        },
+        {
+            "type": "shell",
+            "label": "Build endoscopy_tool_tracking_distributed",
+            "command": "./run",
+            "args": ["build", "endoscopy_tool_tracking_distributed", "--type", "debug"],
+            "options": {
+                "cwd": "${env:WORKSPACE_DIR}"
+            },
+            "group": "build",
+            "problemMatcher": [],
+            "detail": "CMake template build task"
+        },
+        {
+            "type": "shell",
+            "label": "Build endoscopy_tool_tracking_distributed (delay 1s)",
```

Comment:
what is the purpose of these delayed build versions?

Reply:
These are for the compound launch profiles, which lets you launch multiple profiles in parallel. 
With these, we ensure that the driver fragment launches first before the worker fragments.

Take the following distributed endoscopy tool tracking launch profile as an example. It launches three fragments in parallel, but we need to ensure that the `video_in` fragment (driver) starts first before the other two. So, we use the `delay` tasks to delay their start-up.
```
{
            "name": "(compound) endoscopy_tool_tracking_distributed/python",
            "configurations": [
                "(debugpy) endoscopy_tool_tracking_distributed/python - video_in fragment",
                "(debugpy) endoscopy_tool_tracking_distributed/python - inference fragment",
                "(debugpy) endoscopy_tool_tracking_distributed/python - viz fragment",
            ],
            "preLaunchTask": "Build basic_networking_ping",
            "stopAll": true
        }
```